### ISSUE TITLE
Thumbnails Missing/Incorrect Thumbnails Downloaded

### ISSUE THREAD

**Issue by zpz5HAU-tgc3fgw2xwr** [meeb/tubesync#180]
Over the past 5 days or so, random thumbnails have been being downloaded as this default YouTube missing thumbnail image. The thumbnails do exist, and are downloaded correctly when I redownload using youtube-dl, so this is probably another problem caused by yt-dlp.

I have also tried redownloading the media from the TubeSync UI a day later, to test if it was some kind of timing issue. I can see that TubeSync throws out the old thumbnail correctly, so it's not using stale data. But then it redownloads the incorrect thumbnail.

![2021-10-28 - The Official Podcast #256 Alec Baldwin — penguinz0](https://user-images.githubusercontent.com/38122413/139475130-3b27c5b5-2c35-400b-ad8c-905f1a0be8d7.jpg)

This really screws up the UI having these grey thumbnails everywhere, I would much rather it not download any thumbnail than this. At least then my Plex UI wouldn't have entire rows of these ugly grey thumbnails.

**Comment by meeb** [comment]
Yeah it's annoying me as well. It is an upstream issue, but I'll see if there's anything I can hack in in the interim to make this a bit less annoying.

**Comment by tcely** [comment]
@meeb This was fixed a while ago, correct?

**Comment by meeb** [comment]
Yes it should have been, I'll close this for now. Thanks.
